src/scopemeasure.cpp

```diff
@@ -1,26 +1,20 @@
 #include "scopemeasure.h"
 
-extern "C" {
-	void* create_rs_scopemeasure(const char* scope_name);
-	void destroy_rs_scopemeasure(void* object);
-	void rs_scopemeasure_stopover(void* object, const char* stopover_name);
-}
-
 namespace newsboat {
 
 ScopeMeasure::ScopeMeasure(const std::string& func)
+	: rs_object(scopemeasure::bridged::create(func))
 {
-	rs_object = create_rs_scopemeasure(func.c_str());
 }
 
 void ScopeMeasure::stopover(const std::string& son)
 {
-	rs_scopemeasure_stopover(rs_object, son.c_str());
+	rs_object->stopover(son);
 }
 
 ScopeMeasure::~ScopeMeasure()
 {
-	destroy_rs_scopemeasure(rs_object);
+	scopemeasure::bridged::destroy(std::move(rs_object));
```

Comment:
Is this explicit `destroy()` necessary? (I would expect the `rust::Box` to drop the object)

Reply:
You're right, it's unnecessary. I guess I'm still thinking in terms of C bindings :) Thanks!

To verify this, I added an `eprintln!` to the `impl Drop for ScopeMeasure` and checked that tests print out the same number of lines both with and without `destroy()`. That's now included in 60bad8d.

Reply:
Sounds good, thank you for verifying!
I dropped the `FsLock` `destroy()` on my PR as well.